src/components/AboutUsSection/style.ts

```diff
@@ -0,0 +1,83 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  justify-items: center;
+  margin-top: 0;
+`;
+
+export const TextArea = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: 825px;
+  text-align: center;
+  margin: 0;
+
+  @media screen and (max-width: 1668px) {
+    max-width: 500px;
+    font-size: 16px;
+    font-weight: 500;
+  }
+ `;
+
+export const DownButonContainer = styled.div`
+  position: absolute;
+  top: 850px;
```

Comment:
Strange value for `top` property - it should not be hard-coded probably. What did you want to achieve?

Reply:
Discussed f2f: let's use positive relative on parent element and then you will be able to use meaningful value for the top, ie. `top: 150px`